doc/manuals/admin/build_source.md

```diff
@@ -42,6 +43,17 @@ commands that require root privilege):
         tar xfvz v1.0.2.tar.gz
         sudo mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include
 
+* Install libwebsocket from sources:
+
+        wget https://github.com/warmcat/libwebsockets/archive/v1.6.3.tar.gz
+        tar xfvz v1.6.3.tar.gz
+        cd libwebsockets-1.6.3/
+        mkdir build
+        cd build
+        cmake ..          # default cmake sets /usr/local as installation path
+        make
+        sudo make install
+
```

Comment:
As long as MY_PREFERED_PATH would be `/usr/local`, it's ok. However, given that it seems that `cmake` uses that installation path by default, I dont' see the point in making the procedure more complex by the introduction of this `-DCMAKE_INSTALL_PREFIX` variable.

To be discussed at skype.